okapi-common/src/main/java/org/folio/okapi/common/Config.java

```diff
@@ -26,6 +27,38 @@ public static String getSysConf(String key, String def, JsonObject conf) {
     }
   }
 
+  /**
+   * Returns string config info from properties and JSON config.
+   * Return the first that is defined and not empty:
+   * key1 in system properties, key2 in system properties,
+   * key1 in conf parameter, key2 in conf parameter.
+   * Otherwise return def.
+   * @param key1 the first property key (JSON key)
+   * @param key2 the second property key (JSON key)
+   * @param def default value (may be null)
+   * @param conf JSON object configuration
+   * @return property value (possibly null)
+   */
+  public static String getSysConf(String key1, String key2, String def, JsonObject conf) {
+    String v = System.getProperty(key1);
+    if (StringUtils.isNotEmpty(v)) {
+      return v;
+    }
+    v = System.getProperty(key2);
+    if (StringUtils.isNotEmpty(v)) {
+      return v;
+    }
+    v = conf.getString(key1);
+    if (StringUtils.isNotEmpty(v)) {
+      return v;
+    }
+    v = conf.getString(key2);
+    if (StringUtils.isNotEmpty(v)) {
+      return v;
+    }
+    return def;
```

Comment:
Would it be safer to use `StringUtils.isNotBlank()`?

Reply:
Yes. Let's just honor postgres_username and avoid extra complexity like this. See https://issues.folio.org/browse/OKAPI-798 .. Should never have had support for an alias like that.